traits/trait_types.py

```diff
@@ -3033,6 +3077,9 @@ class Map(TraitType):
             trait attribute.
     """
 
+    #: The default value type to use.
+    default_value_type = DefaultValue.constant
```

Comment:
i'm not entirely sure why `Map` and `PrefixMap` have `default_value_type` as `DefaultValue.constant`.

Reply:
Can you elaborate? I can't think of any other default value type that would be appropriate. (Note that the value of a `Map` trait is not a map or a dict - it's a single value.)

```
>>> from traits.api import *
>>> class A(HasTraits):
...     foo = Map({"yes": True, "no": False})
... 
>>> a = A()
>>> a.foo
'yes'
```

Reply:
ohh. never mind. i think i understand now.

Reply:
FWIW, I had to think about this several times, too; the difference in meaning between `Dict` and `Map` is definitely jarring.